aodncore/pipeline/handlerbase.py

```diff
@@ -706,8 +706,8 @@ def _after_state_change(self):
         if self.celery_task is not None:
             self.celery_task.update_state(state=self.state)
 
-    def _file_update_callback(self, name, message=None):
-        self.logger.info("file: '{name}' {message}".format(name=name, message=message))
+    def _file_update_callback(self, **kwargs):
+        self.logger.info("file: '{name}' type: '{type}' message: {message}".format(**kwargs))
```

Comment:
Personally I'd like to see these log messages get _shorter_, not longer. Can we make the format simply
`"{name}{type}: {message}"`? And `type` could be " (deletion)", or just "" for an addition.

Reply:
Sounds good to me. I'm not attached to the format, but we definitely need to make it less ambiguous with additions/deletions. I like the optional suffix idea.

My concern is that a super minimal is a) too ambiguous still (which this seeks to alleviate), and b) gives you nothing sensible to filter on.

From the perspective of something trying to get good information out of the log, possibly in an automated way, this doesn't give you much to filter on. You don't want to be trying to filter on things like ": updated" to match file update lines!

```
2018-06-07 10:48:53,950 SYSINFO [root] DummyHandler transitioned to state: HANDLER_PROCESSED
2018-06-07 10:48:53,950 INFO [root] TestDummyHandler0lbYYu.nc: updated {'archive_path': u'DUMMY/tmp/TestDummyHandlerVyDnqY/DummyHandlerVy9ZVb/collection/TestDummyHandler0lbYYu.nc'}
2018-06-07 10:48:53,951 INFO [root] TestDummyHandler0lbYYu.nc: updated {'is_archived': True}
2018-06-07 10:48:53,951 INFO [root] TestDummyHandler0lbYYu.nc: updated {'publish_type': 'ARCHIVE_ONLY'}
2018-06-07 10:48:53,951 INFO [root] TestDummyHandler0lbYYu.nc: updated {'archive_path': 'NO_PIPELINE/TestDummyHandler0lbYYu.nc'}
2018-06-07 10:48:53,952 INFO [root] TestDummyHandler0lbYYu.nc: updated {'is_archived': True}
2018-06-07 10:48:53,952 INFO [root] TestDummyHandler0lbYYu.nc: updated {'dest_path': 'DUMMY/path/renamed-from-handler-method-/tmp/TestDummyHandlerVyDnqY/DummyHandlerVy9ZVb/collection/TestDummyHandler0lbYYu.nc'}
2018-06-07 10:48:53,952 INFO [root] TestDummyHandler0lbYYu.nc: updated {'is_overwrite': False}
2018-06-07 10:48:53,952 SYSINFO [root] get_harvester_runner -> 'TalendHarvesterRunner'
```

The lines need to be somewhat self describing, because when processing them line by line, you can't rely on the surrounding context.

How about something like this:
```
2018-06-07 11:09:11,194 SYSINFO [root] DummyHandler transitioned to state: HANDLER_PROCESSED
2018-06-07 11:09:11,195 INFO [root] updated file 'TestDummyHandler9HJpt0.nc': {'archive_path': u'DUMMY/tmp/TestDummyHandler_YJkVE/DummyHandlerhIDIOS/collection/TestDummyHandler9HJpt0.nc'}
2018-06-07 11:09:11,195 INFO [root] updated file 'TestDummyHandler9HJpt0.nc': {'is_archived': True}
2018-06-07 11:09:11,196 INFO [root] updated file 'TestDummyHandler9HJpt0.nc': {'publish_type': 'ARCHIVE_ONLY'}
2018-06-07 11:09:11,196 INFO [root] updated file 'TestDummyHandler9HJpt0.nc': {'archive_path': 'NO_PIPELINE/TestDummyHandler9HJpt0.nc'}
2018-06-07 11:09:11,196 INFO [root] updated file 'TestDummyHandler9HJpt0.nc': {'is_archived': True}
2018-06-07 11:09:11,197 INFO [root] updated file 'TestDummyHandler9HJpt0.nc': {'dest_path': 'DUMMY/path/renamed-from-handler-method-/tmp/TestDummyHandler_YJkVE/DummyHandlerhIDIOS/collection/TestDummyHandler9HJpt0.nc'}
2018-06-07 11:09:11,197 INFO [root] updated file 'TestDummyHandler9HJpt0.nc': {'is_overwrite': False}
2018-06-07 11:09:11,197 SYSINFO [root] get_harvester_runner -> 'TalendHarvesterRunner'
```